server/aws-lsp-codewhisperer/src/language-server/codeWhispererServer.ts

```diff
@@ -569,11 +568,11 @@ export const CodewhispererServerFactory =
                 k => params.completionSessionResult[k].accepted
             )
             const acceptedSuggestion = session.suggestions.find(s => s.itemId === acceptedItemId)
-
-            if (acceptedSuggestion !== undefined) {
+            if (acceptedSuggestion !== undefined && acceptedSuggestion.insertText) {
```

Comment:
nit: we can simplify this to `if (acceptedSuggestion?.insertText) ` and remove the inner if clause